packages/components/buttons/primary-button/src/primary-button.tsx

```diff
@@ -93,6 +94,13 @@ const PrimaryButton = <TStringOrComponent extends ElementType = 'button'>(
     disabled: props.isDisabled,
   };
 
+  useWarnDeprecatedProp(
+    !Boolean(props.size === 'small'),
+    'props.size = small',
+    'PrimaryButton',
+    'Please use `medium` or `big` size properties instead.'
+  );
```

Comment:
Not sure if this hook is the best one to use here. I guess the message it will log would be something like:
```
"props.size = small" property of "PrimaryButton" component has been deprecated and will be removed in the next major version. Please use `medium` or `big` size properties instead.
```

Perhaps we can use the `useWarning` hook in this case for a tailored message:
```
useWarning(
  !Boolean(props.size === 'small'),
  'Warning: PrimaryButton `small` value for `size` property has been renamed to `medium`. Please update that value when using this component'
);
```

Reply:
Ah, yes definitely true it looks better now -  https://github.com/commercetools/ui-kit/pull/2443/commits/055ea70fcd3d718c977bf48f1ba760007b6ed0c0

Thank you! 🙇‍♀️ 
![Screenshot 2023-03-15 at 08 53 19](https://user-images.githubusercontent.com/52276952/225243380-c66d8abe-6efb-418a-b720-11efd578923d.png)